pkg/operator/apiserver/audit/manifests/base-policy.yaml

```diff
@@ -6,8 +6,11 @@
     omitStages:
     - "RequestReceived"
     rules:
-    # Don't log requests for events
-    - level: None
+    # Log events at the metadata level. This is necessary to identify
+    # misconfigured clients. They could either spam mutating requests on Events
+    # or list all the Events in the cluster which are two operations that can
+    # lead to disruption.
+    - level: Metadata
```

Comment:
I think we could simply remove it. There is [a catch-all rule](https://github.com/openshift/library-go/blob/8052e81a61efbeabcd1ba0e660d9777a93fc6859/pkg/operator/apiserver/audit/manifests/default-rules.yaml#L10) which would be applied